quickwit/quickwit-metastore/src/metastore/postgresql_metastore.rs

```diff
@@ -259,23 +259,24 @@ fn write_sql_filter<V: Display>(
     sql: &mut String,
     field_name: impl Display,
     filter_range: &FilterRange<V>,
+    value_formatter: impl Fn(&V) -> String,
```

Comment:
Nitpick: Using an enum is a bit better to avoid a typo next time we handle a filter that needs a conversion.

(if you are not aware of nitpick meaning, see: https://github.com/quickwit-oss/quickwit/blob/main/CODE_STYLE.md)